egs2/fleurs/asr1/README.md

```diff
@@ -1,5 +1,70 @@
 <!-- Generated by scripts/utils/show_asr_result.sh -->
-# RESULTS
+# Multilingual ASR - Self-supervised learning features [HuBERT_large_ll60k, Transformer, utt_mvn, 6500 BPE](conf/train_asr_hubert_large_ll60k_transformer.yaml)
+
+## Environments
+- date: `Sun Aug 21 15:18:30 EDT 2022`
+- python version: `3.8.6 (default, Dec 17 2020, 16:57:01)  [GCC 10.2.0]`
+- espnet version: `espnet 202205`
+- pytorch version: `pytorch 1.8.1+cu102`
+- Git hash: `45e8cb9173a072f85ee7a7ccbcae06af7c5c484a`
+  - Commit date: `Wed Jun 1 14:21:14 2022 +0900`
+- Pretrained Model: https://huggingface.co/espnet/wanchichen_fleurs_multilingual_asr_hubert_frontend
+
+### WER
+
```

Comment:
How about adding the language recognition accuracy here?

Reply:
Since the best configs are different between multilingual and monolingual, which one would you like me to symlink to  `conf/train_asr.yaml` ?

Reply:
How about making English one as default?
Also, put both config information in `run.sh`
Then, we should not have confusions, I believe.